### PR TITLE
reference/sql: add declaration for utf8 checking

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? 

This PR adds a declaration for character checking on the `utf8` charset, and the method for skipping the check.

MySQL 5.7 has no check for the character but MySQL 8.0 has. This could make confusion. So the action of TiDB is declared.

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1747

### Which version does your change affect? 

`dev`, `v2.1`, `v3.0`

### CODE REVIEW

**Comment by yikeke** [comment]
Please help review this PR, thx~ @juliezhang1112

**Comment by juliezhang1112** [comment]
LGTM

File: dev/reference/sql/character-set.md [hunk 1]
```diff
@@ -207,4 +207,10 @@ You can use the following statement to specify a particular collation that is re
     SET collation_connection = @@collation_database;
     ```
 
+## Validity check of characters
+
+For the specified `utf8` or `utf8mb4` character set, TiDB only supports the valid `utf8` character, and reports the `incorrect utf8 value` error when the character is invalid. This validity check of characters in TiDB is compatible with MySQL 8.0 but incompatible with MySQL 5.7 or earlier versions.
+
+To mute this error reporting, use `set @@tidb_skip_utf8_check=1;` to skip the character check.
```
Comment (yikeke) [inline dev/reference/sql/character-set.md]:
```suggestion
To disable this error reporting, use `set @@tidb_skip_utf8_check=1;` to skip the character check.
```
"Disable/turn off the error reporting" is more commonly used if you search it in Google.

**Comment by lilin90** [comment]
@TomShawn Please address the conflicts.

**Pull request merged**